client/src/constants/forms.js

```diff
@@ -40,30 +24,25 @@ debug.enabled = false;
  *
  * The `title` is the title that will be shown to the user when they look
  * at that form. It is also used in navigation.
- *
- * `model_key` is what is searched for in the `backend_constants` file to
+ * 
+ * `modelKey` is what is searched for in the `backend_constants` file to
  * retrieve the model that is currently chosen for the form. If there is no
  * model for the form, this property can be left out.
- *
- * `hmModelFlatParamsKey` is what is used as the params key that is sent to
+ * 
+ * `coreParams` lists the parameters associated with the subform common
+ * to all model selections associated with that subform.
+ * 
+ * `paramsKey` is what is used as the params key that is sent to
  * the server.
  *
- * `hmModelParamsKey` is what is searched for in the `backend_constants` file to
- * retrive properties for the different model selections. This is sometimes
- * different than what is sent to the server.
- *
  * @typedef FormDetails
  * @type {{
  *  id: string,
  *  title: string,
- *  model_key: string | undefined,
+ *  modelKey: string | undefined,
+ *  coreParams: String[]
```

Comment:
`coreParams` looks like it can be a `string[]` or `undefined`.

Reply:
:heavy_check_mark: